easybuild/easyconfigs/i/iomkl/iomkl-2017.01.eb

```diff
@@ -0,0 +1,25 @@
+easyblock = "Toolchain"
+
+name = 'iomkl'
+version = '2017.01'
+
+homepage = 'http://software.intel.com/en-us/intel-cluster-toolkit-compiler/'
+description = """Intel Cluster Toolchain Compiler Edition provides Intel C/C++ and Fortran compilers, Intel MKL & OpenMPI."""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+compver = '2017.1.132'
+gcc_maj_min = '5.4'
+gccver = '%s.0' % gcc_maj_min
+binutilsver = '2.26'
+gccsuff = '-GCC-%s-%s' % (gccver, binutilsver)
+dependencies = [
+    ('GCCcore', gccver),
+    ('binutils', binutilsver, '-GCCcore-%s' % gccver),
```

Comment:
@akesandgren any reason you're listing these explicitly? they will get pulled in via `icc` and `ifort` already, and listing them both may confuse people? the other `iomkl` don't list these either